backend/tests/integration/records/serializers/json/test_literature.py

```diff
@@ -673,93 +672,36 @@ def test_literature_detail_serialize_experiment_with_referenced_record(
     record = create_record("lit", data=data)
     experiment_data = {
         "accelerator": {"value": "Accelerator"},
-        "control_number": 1_118_472,
+        "control_number": 1_110_623,
         "institutions": [{"value": "Institute"}],
         "experiment": {"value": "Experiment"},
     }
     create_record_factory("exp", data=experiment_data)
     expected_experiment_data = [
         {
-            "name": "LIGO",
+            "name": "Institute-Accelerator-Experiment",
```

Comment:
we should add a test with linked experiments in literature

Reply:
We test this in this exact test, and in every other test I updated.

Reply:
but for literature this is the only one that tests the experiments linked record and it doesn't test the `$ref`